varya/assets/css/variables-editor.css

```diff
@@ -16,43 +17,43 @@
 body {
 	/* Globals */
 	/* Font Family */
-	--global--font-primary: sans-serif;
-	--global--font-secondary: serif;
+	--global--font-primary: 'Playfair Display', Georgia, Times, serif;
+	--global--font-secondary: 'Fira Sans', Helvetica, Arial, sans-serif;
 	--global--font-code: monospace, monospace;
-	--global--font-ui: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	--global--font-ui: var(--global--font-secondary);
 	/* Font Size */
 	--global--font-size-root: 18px;
 	--global--font-size-ratio: 1.2;
 	--global--font-size-base: 1em;
-	--global--font-size-xs: calc(var(--global--font-size-base) / var(--global--font-size-ratio) / var(--global--font-size-ratio));
-	--global--font-size-sm: calc(var(--global--font-size-base) / var(--global--font-size-ratio));
-	--global--font-size-md: calc(var(--global--font-size-base) * var(--global--font-size-ratio));
-	--global--font-size-lg: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
-	--global--font-size-xl: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
-	--global--font-size-xxl: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
-	--global--font-size-xxxl: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
-	--global--font-size-xxxxl: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
+	--global--font-size-xs: 14px;
+	--global--font-size-sm: 18px;
+	--global--font-size-md: 20px;
```

Comment:
Super close here, but the `font-size-md` variable needs to align with the base font size (18px). Otherwise, the entry-meta and the body copy will be the same font-size, when it should be smaller: 

![image](https://user-images.githubusercontent.com/709581/77805371-1a5da880-7058-11ea-8d27-0e67bb64c544.png)

If we make `font-size-sm` 15px, we can push all the other size up one level, so the max is 64px. We’ll probably have to drop the `xxxxl` 72px rule and only use that for the Block or Heading that actually uses it.